File: solutions/clojure/002.clj
(defn solution
  []
  (let
    [sum 0]
    (loop
      [nl 1 n 2 sum 0]
      (if (< n 4e6)
        (recur n (+ nl n) (if (= (mod n 2) 0) (+ sum n) sum))
        sum))))

(println (solution))

File: solutions/python/002.py
def solution():
    sum = 0
    nl = 1
    n = 2
    while (n < 4e6):
        if (n % 2 == 0):
            sum += n
        new = nl + n
        nl = n
        n = new
    return sum

if __name__ == "__main__":
    print(solution())
